### PR TITLE
Initiate Frontend v2 & Integrate storybook

### PR DESCRIPTION
**Pull request by leo-esaki**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -1,60 +1,59 @@
 {
-  "name": "dashboard",
-  "version": "0.0.1",
+  "name": "sa-dashboard",
```
Comment (justinanderson) [inline frontend/package.json:2]: Please set this to `"atlas-of-opportunity"`. Atlas v2 is going to be used in more places than v1, so I'd like to reduce references to South Australia in the project's name.

File: frontend/package.json [hunk 2]
```diff
@@ -1,60 +1,59 @@
...
     ]
   },
   "browserslist": {
     "production": [
       ">0.2%",
       "not dead",
-      "not op_mini all",
-      "not chrome < 51",
-      "not safari < 10"
+      "not op_mini all"
```
Comment (justinanderson) [inline frontend/package.json:51]: Update browserslist settings to match [Mapbox GL's suggestion](https://docs.mapbox.com/mapbox-gl-js/guides/install/#targeting-transpilation-to-es6-with-browserslist). That means adding:
```
"not ie 11",
"not chrome < 51",
"not safari < 10"
```

File: frontend/.gitignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (justinanderson) [inline frontend/.gitignore:23]: Please merge /frontend/.gitignore into /.gitignore. I'd prefer to have only one .gitignore file for this repo.

**Review by justinanderson — CHANGES_REQUESTED** [review]
In addition to the line comments, please copy over the PDFs from frontend-old/public/papers to frontend/public/papers.

Also, I'm not sure if we'll end up using Sass for this project, but if you and @torosyaneric think it's needed, go for it.

**Comment by leo-esaki** [comment]
@justinanderson Fyi, storybook-examples folder will be removed after we have several components with stories added.

**Review by justinanderson — APPROVED** [review]

**Pull request merged**